src/migrations/20231025135316-add-to-created-via-enum.js

```diff
@@ -0,0 +1,19 @@
+const { GOAL_CREATED_VIA } = require('../constants');
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      return Promise.all(Object.values(GOAL_CREATED_VIA).map((method) => queryInterface.sequelize.query(`
```

Comment:
Could you put this in the migration lib instead, along with the down version. That way we can reuse it for subsequent enum updates.

Reply:
I don't have a down, but I will add this to the migration lib